### ISSUE TITLE
Offline scraper import does not work with 'smb:' paths

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maloep/romcollectionbrowser#213]
```
What steps will reproduce the problem?
1. Use offline scraper for NES
2. Configure rom collection browser to use it's own offline NES.txt
3. Try to import games from smb share

What is the expected output? What do you see instead?
I expect to get my games imported.
But because the paths are prefixed with smb: it's impossible to check if the 
file is a zipfile and no crc could be created, which results in no games being 
imported.

As workaround I have modified the dbupdate.py to replace the 'smb:' with '' in 
the filename before checking the file as zip.


What version of the product are you using? On what operating system?
2.1.2 on windows


Please provide any additional information below.
Logging:
12:56:24 T:3688  NOTICE: RCB_INFO: Rom path: 
['smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/*.zip']
12:56:24 T:3688  NOTICE: RCB_INFO: Reading rom files
12:56:24 T:3688  NOTICE: RCB_INFO: walkDownPath romPath: 
smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/*.zip
12:56:24 T:3688  NOTICE: RCB_INFO: Running walkdown on: 
smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/*.zip
12:56:24 T:3688  NOTICE: RCB_INFO: Begin getFilesByWildcard. pathName = 
smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/*.zip
12:56:24 T:3688  NOTICE: RCB_INFO: dirname: 
smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms
12:56:24 T:3688  NOTICE: RCB_INFO: filemask: *.zip
12:56:24 T:3688  NOTICE: WakeOnAccess [192.168.0.100] trigged by accessing : 
smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms
12:56:24 T:3688  NOTICE: WakeOnAccess success exit, server already running
12:56:24 T:3688  NOTICE: RCB_INFO: xbmcvfs dirs: []
12:56:24 T:3688  NOTICE: RCB_INFO: xbmcvfs files: ['Super Mario World (Europe) 
(Rev A).zip']
12:56:24 T:3688  NOTICE: RCB_INFO: files after walkDown = 
['smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/Super Mario World 
(Europe) (Rev A).zip']
12:56:24 T:3688  NOTICE: RCB_INFO: Files read: 
[u'smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/Super Mario World 
(Europe) (Rev A).zip']
12:56:24 T:3688  NOTICE: RCB_INFO: Start building file dict
12:56:24 T:3688  NOTICE: RCB_INFO: current rom file: 
smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/Super Mario World 
(Europe) (Rev A).zip
12:56:24 T:3688  NOTICE: RCB_INFO: gamename (file): Super Mario World (Europe) 
(Rev A).zip
12:56:24 T:3688  NOTICE: RCB_INFO: gamename (friendly): Super Mario World 
(Europe) (Rev A)
12:56:24 T:3688  NOTICE: RCB_INFO: checkRomfileIsMultirom. gamename = Super 
Mario World (Europe) (Rev A), lastgamename =
12:56:24 T:3688  NOTICE: RCB_INFO: searchGameByCRC = True
12:56:24 T:3688  NOTICE: RCB_ERROR: Error while creating crc: [Errno 22] 
invalid mode ('rb') or filename: 
'smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/Super Mario World 
(Europe) (Rev A).zip'
12:56:24 T:3688  NOTICE: RCB_INFO: Adding crc to crcOfFirstGame-dict: super 
mario world (europe) (rev a): 000000
12:56:24 T:3688  NOTICE: RCB_DEBUG: Add filename 
"smb://192.168.0.100/Games/Emulators/Nintendo_SNES/Roms/Super Mario World 
(Europe) (Rev A).zip" with key "000000"
12:56:24 T:3688  NOTICE: RCB_INFO: start parsing with multi game scraper: SNES


```

Original issue reported on code.google.com by `andy.cor...@gmail.com` on 30 Mar 2015 at 11:51

**Comment by bruny** [comment]
Possible duplicate of closed issue #133

**Comment by maloep** [comment]
Fixed with https://github.com/maloep/romcollectionbrowser/commit/525734937162e12d41df81ecd41dcbf9a0db3d35
